crates/uv/src/requirements.rs

```diff
@@ -79,6 +80,18 @@ impl RequirementsSource {
     }
 }
 
+impl std::fmt::Display for RequirementsSource {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Editable(path) => write!(f, "-e {path}"),
+            Self::RequirementsTxt(path) | Self::PyprojectToml(path) => {
+                write!(f, "{}", path.display())
```

Comment:
Oops, it should actually be `.normalized_display`

Reply:
Ugh lol

Reply:
What's different? Windows?